js/libs/core.js

```diff
@@ -14,14 +14,14 @@ Fliplet.FormBuilder = (function() {
   }
 
   return {
-    on: function(eventName, fn) {
-      eventHub.$on(eventName, fn);
+    on: function(eventName, fn, id) {
+      eventHub.$on(eventName + (id || ''), fn);
```

Comment:
Same for `off` and `emit`.

```suggestion
      eventHub.$on(eventName, fn);
```

Reply:
Done